soc/arm/nuvoton_npcx/Kconfig

```diff
@@ -163,4 +163,20 @@ endif # NPCX_HEADER
 # Select SoC Part No. and configuration options
 source "soc/arm/nuvoton_npcx/*/Kconfig.soc"
 
+config SOC_POWER_MANAGEMENT
+	bool "System Power Management in NPCX family"
+	depends on SOC_FAMILY_NPCX
+	help
+	  This option enables the board to implement SoC specific power
```

Comment:
SoC-specific

Reply:
> Please can you use lower-case hex consistently?

I have found two upper cases that are introduced in the first NPCX driver PR and fixed them. So far, we check the lower case of hex value manually. Do you have any suggestions to check it efficiently? For example, use a script to check the PR's difference? Thanks.

Reply:
I don't know of anything, sorry.

I suspect you could find them with grep, something like

grep 0x[0-9A-Za-z]+

and then put that through

grep [A-Z]

Reply:
@sjg20   Thanks for the advice. I'll try it before submitting PR.